docs/source/development/doc-overview.rst

```diff
@@ -0,0 +1,70 @@
+Contributing to Documentation
+=============================
+
+Using Sphinx
+------------
+IMAP uses `Sphinx <https://www.sphinx-doc.org/en/master/index.html>`_ to generate and maintain documentation. Sphinx allows for automatic usage of Python docstrings, as well features for hierarchical structure, automatic code highlighting, and different output formats.
+
+To add a new documentation file, create a new ``.rst`` file under :file:`docs/source`. RST stands for Restructured Text, which is a type of rich text format used by Sphinx. It is a flexible and powerful format which allows for lots of features which are useful in documentation.
```

Comment:
Not sure if this would fix itself after merging, but the `:file:docs/source` link doesn't seem to do anything?

Reply:
Yes, that fixes itself after merging. I checked all the weird RST specific tags locally. I don't think github is smart enough to make them work.